backend/app/helpers/spree/admin/navigation_helper.rb

```diff
@@ -211,7 +211,7 @@ def configurations_sidebar_menu_item(link_text, url, options = {})
         is_active = url.ends_with?(controller.controller_name) ||
                     url.ends_with?("#{controller.controller_name}/edit") ||
                     url.ends_with?("#{controller.controller_name.singularize}/edit")
-        options.merge!(class: is_active ? 'active' : nil)
+        options.merge!(class: is_active ? 'selected' : nil)
```

Comment:
Use `options[:class] = is_active ? 'selected' : nil` instead of `options.merge!(class: is_active ? 'selected' : nil)`.